### ISSUE TITLE
Can not write data

### ISSUE THREAD

**Issue by dimiii** [palkan/influxer#30]
### Bug report

__System info:__ [Influxdb v1.2.0, influxdb-ruby v0.3.15, Influxer v1.0.0]

__Steps to reproduce:__

1. create data.rb:
```
class Data < Influxer::Metrics
  set_series :data
  set_retention_policy :yearly

  attributes :value
end 
```
2. 3.times { |i| Data.write(value: i) }

__Expected behavior:__ Data stored in database

__Actual behavior:__ Influx responses with 2xx, Influxer reports on successful storing, but there is no data in database

__Additional info:__ 
The old way of storing does not work:
```
$ curl -XPOST "http://localhost:8086/write?db=powerman" -d 'yearly.data value=1'
$ curl "http://localhost:8086/query?db=powerman&q=select+*+from+yearly.data"
{"results":[{"statement_id":0}]}
```

Influx expects retention policy is passed as a parameter (&rp=yearly):
```
$ curl -XPOST "http://localhost:8086/write?db=powerman&rp=yearly" -d 'data value=1'
$ curl "http://localhost:8086/query?db=powerman&q=select+*+from+yearly.data"
{"results":[{"statement_id":0,"series":[{"name":"data","columns":["time","value"],"values":[["2017-07-27T16:24:41.106977621Z",1]]}]}]}
```
See also:
https://github.com/influxdata/influxdb-ruby/blob/master/lib/influxdb/query/core.rb#L67
https://github.com/palkan/influxer/blob/master/lib/influxer/metrics/metrics.rb#L117

**Comment by stasbovanenko** [comment]
Made a quick morning fix d20be75e7b4ffdc1d3e79522112166c710636151. 
Please try Influxer from my rep: https://github.com/stasbovanenko/influxer 
Simpliest way is to add `, github: 'stasbovanenko/influxer'` after `gem 'influxer'` in your Gemfile

**Comment by palkan** [comment]
@stasbovanenko Thanks for the report! Working on it

**Comment by palkan** [comment]
Fixed and published as v1.1.0
